File: beginners/easy/012.efficiency-of-the-carnot-engine/206.saadman-galib.efficiencyOfTheCarnotEngine.cpp
#include<iostream>
#include<iomanip>
using namespace std;

int main(){
    double ef, t1, t2;
    t1 = 373.0;
    t2 = 273.0;
    ef = (1 - (t2 / t1)) * 100;
    cout << "Efficiency of the Carnot's engine is " << fixed << setprecision(2) << ef << "%";
    return 0;
}

File: beginners/easy/012.efficiency-of-the-carnot-engine/206.saadman-galib.efficiencyOfTheCarnotEngine.go
package main 
import ("fmt";
	   "math")

func main(){
	var ef, t1, t2 float64
	t2 = 273.0;
	t1 = 373.0;
	ef = (1 - (t2 / t1)) * 100
	fmt.Println("Efficiency of the Carnot's engine is ", math.Round(ef*100)/100, "%")

}